### PR TITLE
Raft tls without actual domains

### PR DESCRIPTION
**Pull request by volodyad**
There is a demo solution to avoid using actual domains flow in deployment, as for me it seems a bit cumbersome to follow. it can be easily avoided with settings certificates SANS in cryptogen config

### CODE REVIEW

**Comment by raftAtGit** [comment]
@volodyad thanks for the PR! 

this is interesting, I wasn't aware `cryptogen` is capable of this. which versions of cryptogen is supporting this?

I'd like to accept this PR but I guess it shouldnt override the original `scaled-raft-tls` sample. using actual domain names is much more suitable for production environments, especially in a mixed setup, spreading over multiple K8S clusters or other environments.

so, can you please move this to another folder, say `scaled-raft-tls-no-actual-domains`?

File: fabric-kube/samples/scaled-raft-tls/extended/network.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 tlsEnabled: true
-useActualDomains: true
+useActualDomains: false
```
Comment (raftAtGit) [inline fabric-kube/samples/scaled-raft-tls/extended/network.yaml]: You can remove this line, as it defaults to false

File: fabric-kube/samples/scaled-raft-tls/network.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 tlsEnabled: true
-useActualDomains: true
+useActualDomains: false
```
Comment (raftAtGit) [inline fabric-kube/samples/scaled-raft-tls/network.yaml]: You can remove this line, as it defaults to false

**Comment by volodyad** [comment]
it exists at least from 1.1 version https://hyperledger-fabric.readthedocs.io/en/release-1.1/commands/cryptogen-commands.html

File: fabric-kube/samples/scaled-raft-tls-no-actual-domains/README.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+A scaled up HL Fabric network with 3 nodes of Raft orderer spanning 2 organizations and 2 peers per organization. 
+Also demonstrates how TLS can be enabled and actual domain names (instead of internal Kubernetes service names) can be used.
```
Comment (raftAtGit) [inline fabric-kube/samples/scaled-raft-tls-no-actual-domains/README.md]: Is this using actual domains? ;)

**Pull request merged**